test/ILLink.RoslynAnalyzer.Tests/RequiresUnreferencedCodeAnalyzerTests.cs

```diff
@@ -278,7 +278,7 @@ public class C
     [RequiresUnreferencedCodeAttribute(""message"")]
     public int M1() => 0;
 
-    int M2 => M1();
+    int M2 => {|#0:M1()|};
```

Comment:
Just to educate myself, could you elaborate on these changes? Im not sure what is different here and why the location is 0. Is it the fact that is a lambda?

Reply:
Realizing I probably shouldn't have checked these in - the gist of it is that instead of using spans, you can specify locations using this syntax. In this case, the 0 indicates that it's location 0, which you can then reference in the VerifyCS line below. There's nothing actually different, it's just a readability change. At the time, I didn't realize that the Span arguments were automatically generated, so we can leave it the way it is for now. I'll revert the changes

Reply:
@jkoritzinsky would know more about this if you have further questions